src/elements/content-sidebar/additional-tabs/AdditionalTabFtuxTooltip.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ * @file Sidebar Additional Tab FTUX tooltip
+ * @author Box
+ */
+
+import * as React from 'react';
+
+import TargetedClickThroughGuideTooltip from '../../../features/targeting/TargetedClickThroughGuideTooltip';
+
+import type { UseTargetingApi } from '../../../features/targeting/types';
+
+import './AdditionalTabFtuxTooltip.scss';
+
+type Props = {
+    children: React.Node,
+    targetingApi: UseTargetingApi,
+    text: string,
+};
+
+const AdditionalTabFtuxTooltip = ({ children, targetingApi, text }: Props) => (
+    <TargetedClickThroughGuideTooltip
```

Comment:
Does this component require the host Element to be wrapped in a new context? Does this affect third-party use-cases? If so, it would be a breaking change.

Reply:
If you're asking about the HTML structure, it would, since the element would go from `<AdditionalTab />` to `<Tooltip><AdditionalTab /></Tooltip`. The data being passed into the sidebar wouldn't change though, since all of the data used for this tooltip is optional. Additional tabs aren't available to third parties, so we don't need to worry about breaking anyone